mobile/uplink.go

```diff
@@ -172,3 +172,15 @@ func safeError(err error) error {
 	}
 	return fmt.Errorf("%v", err.Error())
 }
+
+// SaltedKeyFromPassphrase returns a key generated from the given passphrase using a stable,
+// project-specific salt
+func (project *Project) SaltedKeyFromPassphrase(passphrase string) (keyData []byte, err error) {
```

Comment:
Should the java code be modified as well?

Reply:
I guess depends if/how we want to expose it to `android-libstorj`